### ISSUE TITLE
Xiaomi Redmi AX6000 - Slow(er) LAN Transfer Speeds

### ISSUE THREAD

**Issue by shauno100** [openwrt/openwrt#11164]
Hi, Not sure if this has been reported already or not, i have noticed an issue even under stock MiWiFi firmware where if i copy a large file from my NAS connected to router at 1Gbps via ethernet, the file transfer rate (MB/s) according to Windows maxs out at approx 90MB/s. I have a spare Xiaomi AX3200 router and the transfer rate maxes out at around 114MB/s. Not a huge difference but i am curious why the transfer rate would be 25-30MB/s less then an older less powerful router. Please let me know what would be required to look into this further. 

I should mention i have the same issue with software/hardware NAT offloading turned on or off and as mentioned the slow(er) transfer rate also occurs with stock firmware.

![Screenshot 2022-11-04 223100](https://user-images.githubusercontent.com/20216406/199973185-d0c8ae76-3180-4e3f-8cbf-d2855b9793ff.png)

**Comment by cryobry** [comment]
Try using iperf for benchmarks, Windows-reported file transfer speeds are not a reliable measurement tool.

**Comment by graysky2** [comment]
Agreed, you can also use `rsync` to move files and get some speed stats in addition to `time`

**Comment by shauno100** [comment]
> Try using iperf for benchmarks, Windows-reported file transfer speeds are not a reliable measurement tool.

Thats fine, I will test further. What I am saying is that when i use any of my other routers in exact same setup file transfer rate is around 115MB/s to my computer from NAS but when i using AX6000 it's never more then 90MB/s so something is clearly different here.

**Comment by zk00790** [comment]
Encountered the same problem
![屏幕截图 2024-11-12 235035](https://github.com/user-attachments/assets/7d50be9e-7e0f-46bd-843b-c420521d52cc)
![屏幕截图 2024-11-12 235049](https://github.com/user-attachments/assets/3d4c0186-671b-484f-925d-059b6e16cf76)

**Comment by shauno100** [comment]
I've moved on and have a different router, i hope you guys have success in getting this looked at by the devs but considering 2 years have passed since i reported the issue i doubt it will ever be resolved.
